src/wandb_osh/syncer.py

```diff
@@ -35,6 +38,25 @@ def __init__(
         self.wait = wait
         self.wandb_options = wandb_options
         self._timeout = timeout
+        self.num_workers = num_workers
+        self.target_queue: Queue = Queue()
+        self.workers: list[Process] = []
+
+    def start(self) -> None:
```

Comment:
Even though the new name makes more sense, this would break backward compatible, so probably best to keep this as "loop".